db/dataGeneration/generateData.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const faker = require('faker');
+
+// Helper functions to generate random data
+let randomName = () => {
```

Comment:
I would change all of these function definitions to constant values. Less of a chance to accidentally overwrite them and create bugs later.